cmd/create.go

```diff
@@ -125,6 +125,21 @@ func expandOrDie(path string) (config.Blueprint, *config.YamlCtx) {
 	// Expand the blueprint
 	checkErr(bp.Expand(), ctx)
 	validateMaybeDie(bp, *ctx)
+
+	// TODO: Remove this warning when v5 deprecation is complete
+	alreadyContainsV5 := false
+	bp.WalkModulesSafe(func(mp config.ModulePath, m *config.Module) {
+		if m.Source == "community/modules/scheduler/schedmd-slurm-gcp-v5-controller" && !alreadyContainsV5 {
```

Comment:
Let's use `Contains(m.Source, "schedmd-slurm-gcp-v5-controller")` to accommodate for multitude of different ways to reference a module.